common-legacy/build.gradle

```diff
@@ -16,6 +16,7 @@ tasks.compileThrift.doLast {
         def content = oldContent
                 .replaceAll('fromRevision', 'from')
                 .replaceAll('toRevision', 'to')
+                .replaceAll('isIsOptional', 'isOptional')
```

Comment:
I don't think we need this because we are adding a new property. There's no need for backward compatibility and we can just use `isIsOptional()` internally. A user is not supposed to use Thrift-generated classes.